js/ai.js

```diff
@@ -67,6 +67,14 @@ export default {
         this.test = 0;
         this.foundSet = false;
 
+        //have ai check each time it resumes if game should end; copied from board logic
+        if (isDev)  console.log(`ai sets found: ${this.findValidSets().length}`);
+        if (isDev)  console.log(`ai cards left: ${deck.stock.length}`);
```

Comment:
suggest making the logging a single logging line with both elements being checked in one.

Reply:
Done in latest commit